README.md

```diff
@@ -1,7 +1,7 @@
 # Hyper-Orama
 [![All Contributors](https://flat.badgen.net/github/contributors/cwlowder/hyper-orama)](#contributors)
 
-A screen recorder for the hyper terminal. It publishes all captured sessions to [now.sh](https://zeit.co/now)
+A screen recorder for the hyper terminal. It publishes all captured sessions to [now.sh](https://zeit.co/now).
```

Comment:
Perhaps we should also mention that it saves videos locally as well?

Reply:
Good call! Can you add that?

Reply:
You can add it yourself directly in the PR! hit ctrl+g